### PR TITLE
Fix edit credential action.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: edit-credential-modal-directive.js [hunk 1]
```diff
@@ -39,7 +38,12 @@ function factory(brAlertService, brCredentialService) {
       }
       model.loading = true;
       brAlertService.clearFeedback();
-      brCredentialService.collection.update(credential)
+      var options = {};
+      var search = $location.search();
+      if('id' in search) {
+        options.url = $location.path() + '?id=' + search.id;
+      }
```
Comment (dlongley) [inline edit-credential-modal-directive.js]: I think this is ok for now, but we should add a TODO that says the update URL should be passed into the directive as an optional param. We don't want to do any sort of magic here.

Comment (dlongley) [inline edit-credential-modal-directive.js]: Actually, if it's not too hard, we should do that now.

File: share-credential-modal.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<br-modal br-title="Share Credential">
+  <div name="br-modal-body">
+    <input type="text" value="{{shareUrl}}" br-select-text="true" size="50"></input><br>
+    Press <strong>Ctrl + C</strong> to copy URL
```
Comment (davidlehn) [inline share-credential-modal.html]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd
Also Ctrl-C is not universal.  For instance on Macs.

Comment (mattcollier) [inline share-credential-modal.html]: Do we want to strike the copy instructions entirely, or add instructions for the Mac as well?

**Pull request closed without merging**